hudi-hive/src/main/java/org/apache/hudi/hive/HoodieHiveClient.java

```diff
@@ -192,7 +192,9 @@ private String getPartitionClause(String partition) {
     String alterTable = "ALTER TABLE " + syncConfig.tableName;
     for (String partition : partitions) {
       String partitionClause = getPartitionClause(partition);
-      String fullPartitionPath = FSUtils.getPartitionPath(syncConfig.basePath, partition).toString();
+      Path partitionPath = FSUtils.getPartitionPath(syncConfig.basePath, partition);
+      String fullPartitionPath = !partitionPath.toUri().getScheme().equals("hdfs") ? partitionPath.toString()
```

Comment:
small nit: can we use the `StorageSchemes.HDFS` enum? this way we can track this special handling over time

Reply:
That make sense. Replaced.